README.md

```diff
@@ -5,6 +5,10 @@
 
 Natron is a free, open-source (GPLv2 license) video compositor, similar in functionality to Adobe After Effects, Foundry's Nuke, or Blackmagic Fusion. It is portable and cross-platform (GNU/Linux, macOS, and Microsoft Windows).
 
+![Hero Image of Natron](./MarkdownAssets/Natron-Hero-Image.png)
```

Comment:
This image would fit reasonably well on the new website, I would personally like to transition the readme more towards development information and less towards more user-facing stuff like this image when the new website comes online but that's just my opinion.

Reply:
Yeah, though you know my reasons for adding it while the new website is still not here. Additionally, there are notable semi-complex C/C++ projects that do this ([example](https://github.com/rxi/lite)) and adding a hero image is going to improve first impressions.

Reply:
Yep, its not out of the question and I won't fight to remove it if other devs do like it.  Would recommend adapting to GitHub's style instead of the dark background though, something simpler like this would probably fit better in a GitHub readme document... Obviously with something open in the application though :P

<img width="1904" alt="Screen Shot 2021-04-29 at 11 50 32 PM" src="https://user-images.githubusercontent.com/5672810/116646354-cf425300-a945-11eb-95af-faf58d86ab7e.png">


My reasoning for wanting to push features and content that users care about to the website is that it means fewer pieces of duplicate content to manage and fewer locations for users to know about where they might find this information.  Transitioning the readme to developer-focused information makes sense as it's on GitHub, transitioning the website to user-focused information makes sense because it isn't!

Reply:
Yeah - understandable, and good points all around. Dark background was intentional though, trying to add a white gradient was super jarring. And yes, the README should be developer focused. However, the designer part of me just keeps trying to visualize everything as a design, and looking at a README without an image just feels empty.

Also, posting the current not-exactly-amazing UI on a simple screenshot is exactly what I wanted to avoid; that's the entire reason behind the gentle fade-to-dark gradient that I spent quite a lot of time tweaking in Krita. There is a reason why big open-source projects often have a README banner or logo - see <https://github.com/Dogfalo/materialize> and <https://github.com/tailwindlabs/tailwindcss>

On a side note, I think the main devs should check out the TravisCI build issue on MacOS. It is preventing a lot of PRs that should be merged, notably [this one](https://github.com/NatronGitHub/Natron/pull/612) (doesn't apply to this PR, this PR *definitely* needs to be reviewed first).

Reply:
Personally, I'm a much bigger fan of logos and banners than I am of large screenshots.  They take up less space and are just as good if not better for branding.

CI isn't a problem with this PR and I'm sure they'll get around to it eventually.

Reply:
Will leave that a decision for the core devs to make